src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -111,7 +104,7 @@ export default class BrowseSkill extends React.Component {
     };
   }
 
-  componentDidMount() {
+  componentDidMount = () => {
```

Comment:
Arrow functions not needed in lifecycle methods

Reply:
@akshatnitd I had put it there for the sake of consistency but I guess you are right, that is unnecessary.